code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -235,3 +235,17 @@
 		return "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life and [t_his] soul has lost the will to live...</span>"
 	//This mob has a ghost linked that could still reenter the body, so the soul only departed
 	return "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life and [t_his] soul has departed, but the link is not yet fully broken...</span>"
+
+///
+/mob/living/carbon/human/proc/update_lips(new_style, new_colour, apply_trait)
+	lip_style = new_lips
```

Comment:
Depressing, please fix so I can speedmerge before mothblocks wakes up

Reply:
its too late...i woke up after a nap....and my day is ruined...

Reply:
NOOOOOOO